driver.sh

```diff
@@ -131,6 +131,10 @@ build_linux() {
     [[ $ARCH != "x86_64" ]] && cat ../configs/tt.config >> .config
     mako_reactor olddefconfig &>/dev/null
   fi
+  # Make sure we build with CONFIG_DEBUG_SECTION_MISMATCH so that the
+  # full warning gets printed and we can file and fix it properly.
+  ./scripts/config -e DEBUG_SECTION_MISMATCH
+  mako_reactor olddefconfig &>/dev/null
```

Comment:
you can remove L132 then, as it's the same as L137 unconditionally now.

Also, why do you do:
1. make defconfig
2. scripts/config -e ...
3. make olddefconfig

I would think 3 is unnecessary?